site/content/component/hono-server.md

```diff
@@ -19,24 +19,74 @@ The following table provides an overview of the configuration options the adapte
+| `HONO_SERVER_INSECURE_PORT_BIND_ADDRESS`<br>`--hono.server.insecurePortBindAddress` | no | `127.0.0.1` | The IP address the insecure port of the Hono server shall bind to. By default the server binds to the *loopback device* address, i.e. the server will **not** be accessible from other hosts. |
+| `HONO_SERVER_INSECURE_PORT_ENABLED`<br>`--hono.server.insecurePortEnabled` | no | `false` | Enables the Hono server to open an insecure port (not secured by TLS). <br>This allows for a dual port setup of the Hono server (secure and insecure port). |
 | `HONO_SERVER_KEY_PATH`<br>`--hono.server.keyPath` | no | - | The absolute path to the PEM file containing the private key the Hono server uses for authenticating to clients. Either this or the `HONO_SERVER_KEY_STORE_PATH` option **must** be set to enable TLS secured connections with clients. |
-| `HONO_SERVER_KEY_STORE_PATH`<br>`--hono.server.keyStorePath` | no | - | The absolute path to the Java key store containing the private key and certificate the Hono server uses for authenticating to clients. Either this or the `HONO_SERVER_KEY_PATH` and `HONO_SERVER_CERT_PATH` options **must** be set to enable TLS secured connections with clients. The key store format can be either `JKS` or `PKCS12` indicated by a `.jks` or `.p12` file suffix. |
 | `HONO_SERVER_KEY_STORE_PASSWORD`<br>`--hono.server.keyStorePassword` | no | - | The password required to read the contents of the key store. |
+| `HONO_SERVER_KEY_STORE_PATH`<br>`--hono.server.keyStorePath` | no | - | The absolute path to the Java key store containing the private key and certificate the Hono server uses for authenticating to clients. Either this or the `HONO_SERVER_KEY_PATH` and `HONO_SERVER_CERT_PATH` options **must** be set to enable TLS secured connections with clients. The key store format can be either `JKS` or `PKCS12` indicated by a `.jks` or `.p12` file suffix. |
 | `HONO_SERVER_MAX_INSTANCES`<br>`--hono.server.maxInstances` | no | *#CPU cores* | The number of verticle instances to deploy. If not set, one verticle per processor core is deployed. |
-| `HONO_SERVER_PORT`<br>`--hono.server.port` | yes | `5672` | The port the server should listen on. |
-| `HONO_SERVER_TRUST_STORE_PATH`<br>`--hono.server.trustStorePath` | no  | - | The absolute path to the Java key store containing the CA certificates the Hono server uses for authenticating clients. The key store format can be either `JKS`, `PKCS12` or `PEM` indicated by a `.jks`, `.p12` or `.pem` file suffix. |
+| `HONO_SERVER_PORT`<br>`--hono.server.port` | no | `5671` | The secure port the server shall listen on. <br> The configured port is guaranteed to be always TLS secured. If the TLS setup is not correct or incomplete, this port will NOT be opened. |
```

Comment:
IMHO it would be better to provide better guidance on what needs to be done instead of saying what won't work, e.g.
> The secure port that the server should listen on. <br> In order for this to work either `HONO_SERVER_KEY_STORE_PATH` (and `HONO_SERVER_KEY_STORE_PASSWORD`) or `HONO_SERVER_KEY_PATH` and `HONO_SERVER_CERT_PATH` need to be set appropriately. Otherwise the Hono server will not open the configured port.

WDYT?